packages/calcite-components/src/components/date-picker/date-picker.tsx

```diff
@@ -124,6 +151,11 @@ export class DatePicker implements LocalizedComponent, LoadableComponent, T9nCom
     this.maxAsDate = dateFromISO(max);
   }
 
+  /**
+   * Specifies if the month abbreviations are used by the component.
+   */
+  @Prop() monthAbbreviations: boolean = false;
```

Comment:
We might be able to use `Intl.DateTimeFormat` for this in the future, so WDYT about changing the prop to:

```tsx
monthStyle = "short" | "long"; // "narrow" could be added later
```

Reply:
i think using monthStyle might give the idea to the user that number format is possible when `monthStyle="short"` (ex: 08 for Aug/August) as per UNICODE specs https://cldr.unicode.org/translation/date-time/date-time-patterns#h.aa5zjyepm6vh.  Additionally in context of calcite  these values might be confused for size of the month.

We can use "abbreviated | wide " and add "narrow" in future. Thoughts?

Reply:
> We can use "abbreviated | wide " and add "narrow" in future. Thoughts?

I dig it!